src/js/Cell.js

```diff
@@ -16,22 +17,41 @@ class Cell extends MazeEntity {
      * @returns {Array.<Cell>} List of neighboring cells
      */
     getNeighbors() {
-
+        return this.getNeighboringEntitiesList(2)
     }
 
     /**
-     * @param {Cell} cell 
+     * @param {Cell} cell
      * @returns {boolean} Represents whether there is a removed wall between cells
      */
     canAccessCell(cell){
+      return (
+          cell.visited==false
+          && this.isNeighboringCell(cell)
+          && this.getInterveningWall(cell).state === WallState.PENDING
```

Comment:
shouldn't this be `!== WallState.CONFIRMED` ?

Reply:
Ohhhhhh hmmm you're probably right? Although lol, our description on the function implies it should be ===REMOVED. I forget exactly how were using this lol

Reply:
oh fuck, and I think you're right - it should only be REMOVED